manual/src/tutorials/moduleexamples.etex

```diff
@@ -130,50 +114,54 @@ Signatures are interfaces for structures. A signature specifies
 which components of a structure are accessible from the outside, and
 with which type. It can be used to hide some components of a structure
 (e.g. local function definitions) or export some components with a
-restricted type. For instance, the signature below specifies the three
-priority queue operations "empty", "insert" and "extract", but not the
-auxiliary function "remove_top". Similarly, it makes the "queue" type
+restricted type. For instance, the signature below specifies the
+queue operations "empty", "add", "top" and "pop", but not the
+auxiliary function "make". Similarly, it makes the "queue" type
 abstract (by not providing its actual representation as a concrete type).
```

Comment:
This hasn't been changed from the old text, but I would change `representation` to `definition` which is a bit less vague.
```suggestion
abstract (by not providing its actual definition as a concrete type).
```

Reply:
Type abstraction used to be called "representation hiding", in that it hides the actual in-memory representation of values of the type.  I kind of like this intuition.

Reply:
OK, makes sense.